### DISCUSSION TITLE
Getting permissions error not sure how to handle these

### DISCUSSION THREAD

**Discussion by Sunraj751** [gis-ops/docker-valhalla#150] [Q&A]
Hi so after running the docker and  I get this following error. I think the issue is it is needs root user privileges for run.sh which unfortunately I can't have access to : 

```
INFO: Running container with user valhalla UID 59999 and GID 59999.
sudo: The "no new privileges" flag is set, which prevents sudo from running as root.
sudo: If sudo is running in a container, you may need to adjust the container configuration to disable the flag.
```

**Comment by nilsnolde** [comment]
Just an error is not enough. What command/docker-compose.json did you use?

Seems it wants to run sudo as root, which shouldn’t be a problem generally but also shouldn’t happen. Since we didn’t change anything in a long time it’s likely it’s something with your setup/system (windows?), otherwise we’d have had many other complaints already.

**Reply by Sunraj751** [reply]
Hi

1. I am trying to run this on azure container apps. It inherently used the docker file (from searching around i found it does not use docker compose).
2. All I did was point to a valhalla docker image and add the url to download from as environment variable (I can attach screenshots of my steps if needed)
3. And with ACA you cannot run stuff as root cause it simply doesn’t allow it. It supports linux containers.

**Reply by nilsnolde** [reply]
Ah azure I heard of problems with before. Sorry, the maintainers can’t support here.

**Reply by Sunraj751** [reply]
Okay can we continue this discussion via email ?

**Reply by nilsnolde** [reply]
Sure, it’s enquiry@gis-ops.com. I’ll send you a proposal for commercial support.

**Reply by AnQueth** [reply]
was there a way to make it work ?

**Reply by Sunraj751** [reply]
> was there a way to make it work ?

Nope tried those things, still didn't work
